### PR TITLE
Add files.

### PR DESCRIPTION
**Pull request by andrazhribernik**

### CODE REVIEW

**Comment by andrazhribernik** [comment]
@trsherborne Could you please review. 

Readme description is not done yet. If you have any good ideas for the main repo description, I would highly appreciate it :)

Please also check inline comments if they make sense. My idea is that master branch will contain `tutorial.ipynb` notebook without a code and branch `tutorial-solution` will contain all the code as well. After a workshop we can swap that, since it is nice to get notebook with all the code.

Do you have any idea for repo name? I am not supper happy with  `nlp-in-10-lines`. Do you think something like `spacy-tutorial` would be better?

**Comment by tomsherborne** [comment]
@andrazhribernik I think the repo name could be `2016-pycon-nlp-in-10-lines` which identifies the repo to visitors and matches the workshop name for consistency. I like the idea of having the solutions on a separate branch you can commit at the end of the workshop, perhaps in front of the audience so they know that it's there.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,24 @@
...
+
+This repository contains a step by step tutorial on some basic Natural Language Processing tasks using Python library
+ [spaCy](http://spacy.io/).
+
+## Get started
+We recommend you to install all the dependencies in virtualenv, however this step could be skipped.
+
+    virtualenv venv
+    source venv/bin/activate
+
```
Comment (tomsherborne) [inline README.md:11]: I can add conda env instructions to this when merged

Comment (andrazhribernik) [reply README.md:11]: Sure!

File: tutorial.ipynb [hunk 2]
```diff
@@ -0,0 +1,530 @@
...
+    "# Process sentences 'Hello, world. Here are two sentences.' using spaCy\n",
+    "\n",
+    "doc = nlp('Hello, world. Here are two sentences.')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Get tokens and sentences"
```
Comment (trsherborne) [inline tutorial.ipynb]: add a 1 liner about what we consider a "token" in NLP/spaCy

File: tutorial.ipynb [hunk 3]
```diff
@@ -0,0 +1,530 @@
...
+    "# Print sentences (one sentence per line)\n",
+    "for sent in doc.sents:\n",
+    "    print(sent)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Part of speech tags"
```
Comment (trsherborne) [inline tutorial.ipynb]: Same as above. Short description of what a "speech tag" is would aid beginners

**Comment by tomsherborne** [comment]
@andrazhribernik I like the notebook and we can work on the README next. I think the readme should include a dictionary of commonly used terms like "noun chunk", "speech tag", "token". I know it can seem basic but if people can read it before they will be able to get more out of the workshop itself

**Comment by tomsherborne** [comment]
@andrazhribernik write access to merge please!

**Pull request merged**